mcux/mcux-sdk/boards/evkmimxrt595/flash_config/flash_config.c

```diff
@@ -5,7 +5,7 @@
  * SPDXLicense-Identifier: BSD-3-Clause
  */
 #include "flash_config.h"
-#include "board.h"
+/* #include "board.h" */
 
```

Comment:
Need to get the SDK team to remove this include

Reply:
This has been brought to the attention of the SDK team to fix.